### ISSUE TITLE
Cannot find Moralis extension on Firebase

### ISSUE THREAD

**Issue by michaelwang11394** [MoralisWeb3/firebase-extensions#4]
Hi, 

The Moralis extension is no longer available on the firebase extension market. How can I use the extension?

I found this on stackoverflow, but didn't work for me.
https://stackoverflow.com/questions/70611848/how-to-install-unofficial-firebase-extension

![image](https://user-images.githubusercontent.com/11284232/215927924-9d0ff837-eeba-4883-a7c4-8299b11911c6.png)

**Comment by b4rtaz** [comment]
It's weird. On my PC it works. Which version of the firebase-tools are you using?

`firebase --version`

**Comment by michaelwang11394** [comment]
> It's weird. On my PC it works. Which version of the firebase-tools are you using?
> 
> `firebase --version`

@b4rtaz thanks for the response! I'm on version `8.3.0`. I see from that there's a version 11 now on [npm](https://www.npmjs.com/package/firebase-tools?activeTab=versions). I tried running npm update to this version, but am still stuck on version 8.3. I then tried uninstalling and reinstalling everything. And am still stuck on version 8.3. 

Is this the root of my problem?
